semgrep-core/src/core/il/IL_lvalue_helpers.mli

```diff
@@ -1,2 +1,3 @@
-val lvar_of_instr_opt : IL.instr -> IL.name option
+val dotted_lvars_of_lval : IL.lval -> IL.name list
```

Comment:
This deserves a comment too I think, it could just be an example input-output pair.